docs/guides/primitives.mdx

```diff
@@ -17,45 +17,40 @@ In this context, a *primitive* is the smallest processing instruction, the simpl
 one can create something useful for a given abstraction level.
 
 The recent progress in quantum computing has increased the need to work at higher levels of abstraction.
-As we move toward larger QPUs (quantum processing units) and more complex workflows, the focus shifts from interacting with individual
+As we move toward larger quantum processing units (QPUs) and more complex workflows, the focus shifts from interacting with individual
 qubit signals to viewing quantum devices as systems that perform tasks we need.
 
 The two most common tasks quantum computers are used for are sampling quantum states and calculating expectation values.
-These tasks motivated the design of the Qiskit primitives: Sampler and Estimator.
+These tasks motivated the design of the Qiskit primitives: **Estimator** and **Sampler**.
+
+- [Estimator](#estimator) computes expectation values of observables with respect to states prepared by quantum circuits.
+- [Sampler](#sampler) samples the output register from quantum circuit execution.
 
 In short, the computational model introduced by the Qiskit primitives moves quantum programming one step closer
 to where classical programming is today, where the focus is less on the hardware details and more on the results
 you are trying to achieve.
 
-## Implementation of Qiskit primitives
-
-The Qiskit primitives are defined by open-source primitive base-classes, from
-which different providers can derive their own Sampler and Estimator implementations. Among the implementations
-using Qiskit, you can find reference primitive implementations for local simulation in the `qiskit.primitives` module.
-Providers like Qiskit Runtime enable access to appropriate QPUs through native implementations of
-their own primitives.
+<Admonition type="note">
+The Qiskit SDK holds the definitions of the `Estimator` and `Sampler` primitives and acts  as the source-of-truth for this concept. It also contains reference implentations [(explained below)](./primitives#implementations) to assist providers of quantum compute resources in developing services which support Qiskit users.
 
-## Benefits of Qiskit primitives
+For users of IBM Quantum hardware, the [`qiskit-ibm-runtime`](./index#qiskit-runtime) package possesses an implementation of these primitives which allow you to execute quantum workloads on IBM Quantum hardware.
```

Comment:
```suggestion
For users of IBM Quantum hardware, the [`qiskit-ibm-runtime`](./index#qiskit-runtime) package possesses an implementation of these primitives which allow you to execute quantum workloads on IBM Quantum&trade; hardware.
```

Reply:
Note: we can now remove the IBM Quantum trademark farther down the page in the Next steps